webcomponentsjs-when-needed.sublime-snippet

```diff
@@ -0,0 +1,23 @@
+<snippet>
+	<content><![CDATA[
+var webComponentsSupported = (
+  'registerElement' in document &&
+  'import' in document.createElement('link') &&
+  'content' in document.createElement('template'));
+
+if (!webComponentsSupported) {
+  var script = document.createElement('script');
+  script.async = true;
+  script.src = 'webcomponents-lite.min.js';
+  script.onload = finishLazyLoading();
+  document.head.appendChild(script);
+} else {
+  finishLazyLoading();
+}
+]]></content>
+	<!-- Optional: Set a tabTrigger to define how to trigger the snippet -->
+	<tabTrigger>webcomponents</tabTrigger>
```

Comment:
Would you mind changing this tab trigger to something like `polyfill`. There's already a tabTrigger for `webcomponents` though it's in html, it's kinda confusing to have both.
https://github.com/robdodson/PolymerSnippets/blob/master/webcomponentsjs.sublime-snippet#L6